main.c

```diff
@@ -20,7 +20,9 @@ int main(int argc, char* argv[]) {
   }
 #endif
 
-  if (strcmp(argv[1], "on") == 0) {
+  if (strcmp(argv[1], "show") == 0) {
+    show();
```

Comment:
Note also that the darwin version of show() actually returns TRUE, which is an error status.